packages/core/src/App/Containers/WarningScamMessageModal/warning-scam-message.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { connect } from 'Stores/connect';
+import { Modal, DesktopWrapper, MobileWrapper } from '@deriv/components';
+import { PropTypes } from 'prop-types';
+import { withRouter } from 'react-router';
+import './warning-scam-message-modal.scss';
+import WarningScamMessageModal from './warning-scam-message-modal';
+
+const WarningScamMessage = ({
+    is_logged_in,
+    client_country,
+    is_dark_mode_on,
+    has_read_scam_message,
+    setScamMessageLocalStorage,
+}) => {
+    // debugger;
+    const [is_message_read, setIsMessageRead] = React.useState(false);
+
+    const acknowledgeMessage = () => setIsMessageRead(!is_message_read);
+
+    /* Check whether the account is logged_in, and client from country 'br' = Brazil, and have not yet read the scam message warning pop up */
+    if (is_logged_in && client_country === 'br' && !has_read_scam_message) {
+        return (
```

Comment:
- You should simplify this condition and return `null` in the end.
- This is the same condition that is being used in `app-modals` so please reuse it by moving it to `ui-store`